content/en/blogs/transform-traditional-applications-into-microservices.md

```diff
@@ -0,0 +1,227 @@
+---
+title: 'Refactor Traditional Applications into Microservices to Enable the Traffic Monitoring Feature'  
+tag: 'KubeSphere, Kubernetes, Microservices'  
+keywords: 'KubeSphere, Kubernetes, Microservices, Traffic Monitoring, Service Mesh'  
+description: 'This article describes how to refactor a traditional application into microservices to use service mesh features, such as grayscale release, traffic monitoring, and tracing.'   
+createTime: '2021-07-07'  
```

Comment:
I prefer to use the date of the English version instead of the original publish date.